Add list of open principles (#247)
OpenAstronomy/openastronomy.github.io: index.md

OpenAstronomy is a collaboration between open source astronomy and astrophysics
projects to share resources, ideas, and to improve code.

## Principles of OpenAstronomy

The core principles of OpenAstronomy include:

> 1. Open Source: OA projects are committed to their software being Open Source. This promotes accessibility, reliability, and scientific reproducibility.

Reviewer: ```suggestion
1. Open Source: OA projects are committed to their software being Open Source[1]. This promotes accessibility, reliability, and scientific reproducibility.
[1] https://opensource.org/docs/osd
```

> 2. Open Development: OA projects support the principle of "open development", a model where the software is both *by* and *for* the community.  More concretely, these projects try to work in publicly-visible spaces, encouraging discussion and participation from their public user community on both code and project-level decisions.

Reviewer 2: ```suggestion
2. Open Development: OA projects support the principle of "open development"[1], a model where the software is both *by* and *for* the community.  More concretely, these projects try to work in publicly-visible spaces, encouraging discussion and participation from their public user community on both code and project-level decisions.
[1] Tollerud, E., “Sustaining Community-Driven Software for Astronomy in the 2020s”, vol. 51, no. 7, 2019.
```

> 2. Open Development: OA projects support the principle of "open development", a model where the software is both *by* and *for* the community.  More concretely, these projects try to work in publicly-visible spaces, encouraging discussion and participation from their public user community on both code and project-level decisions.

Author: ```suggestion
2. Open Development: OA projects support the principle of Open Development, a model where the software is both *by* and *for* the community.  More concretely, these projects try to work in publicly-visible spaces, encouraging discussion and participation from their public user community on both code and project-level decisions.
```

> 2. Open Development: OA projects support the principle of "open development", a model where the software is both *by* and *for* the community.  More concretely, following open development means making code and project-level decisions mainly in publicly-visible spaces. These projects are also open to input from their public user community for these same decisions.

Reviewer 3: Would a link/citation to the white paper make sense here?

> 2. Open Development: OA projects support the principle of "open development", a model where the software is both *by* and *for* the community.  More concretely, following open development means making code and project-level decisions mainly in publicly-visible spaces. These projects are also open to input from their public user community for these same decisions.

Reviewer: I like by and for the community, if you definition of "community" is broad enough it covers pretty much everything. For me it's really "*by* and *for* the [user] community".

> 2. Open Development: OA projects support the principle of "open development", a model where the software is both *by* and *for* the community.  More concretely, following open development means making code and project-level decisions mainly in publicly-visible spaces. These projects are also open to input from their public user community for these same decisions.

Reviewer 4: I'm still not sure, it doesn't necessarily scale well. Small projects might have concrete needs as primary priorities to address, but not much resource to support, review and take on the wider community pressure. Yet, I think these projects, too can operate within the open development framework.

> 2. Open Development: OA projects support the principle of "open development", a model where the software is both *by* and *for* the community.  More concretely, following open development means making code and project-level decisions mainly in publicly-visible spaces. These projects are also open to input from their public user community for these same decisions.

Author: ```suggestion
2. Open Development: OA projects support the principle of "open development", a model where the software is both *by* and *for* the community.  More concretely, these projects are also open to input on both code and project-level discussions from their public user community.
```

> 2. Open Development: OA projects support the principle of "open development", a model where the software is both *by* and *for* the community.  More concretely, following open development means making code and project-level decisions mainly in publicly-visible spaces. These projects are also open to input from their public user community for these same decisions.

Author: ```suggestion
2. Open Development: OA projects support the principle of "open development", a model where the software is both *by* and *for* the community.  More concretely, these projects try to work in publicly-visible spaces and encourage discussion and participation from their public user community on both code and project-level decisions.
```

> 2. Open Development: OA projects support the principle of "open development", a model where the software is both *by* and *for* the community.  More concretely, following open development means making code and project-level decisions mainly in publicly-visible spaces. These projects are also open to input from their public user community for these same decisions.

Reviewer: ```suggestion
2. Open Development: OA projects support the principle of "open development", a model where the software is both *by* and *for* the community.  More concretely, these projects try to work in publicly-visible spaces, encouraging discussion and participation from their public user community on both code and project-level decisions.
```

> 3. Open Science: OA projects support open and accessible data and reproducible science both through good data management practices and effective open software.

Reviewer 4: how far should this go? a pipeline/part of a pipeline, or a generic but highly reusable tool primarily developed for a survey with proprietary data period would fail here?

Reviewer 3: I'm also in favor of leaving data out of the requirements conversation. I think that might touch a nerve that triggers greedy skepticism.

Reviewer: The software can "support open and accessible" and be detached from data which has proprietary periods etc. I do agree that this is a fine and more controversial line than the software.

OpenAstronomy is a collaboration between open source astronomy and astrophysics
projects to share resources, ideas, and to improve code.

## Principles of OpenAstronomy

The core principles of OpenAstronomy (OA) include:

1. Open Source: OA projects are committed to their software being Open Source[1]. This promotes accessibility, reliability, and scientific reproducibility.

> [1] https://opensource.org/docs/osd

Reviewer 4: move references to the bottom, and make numbering continual (e.g. not two [1], but a [1] and a [2]

2. Open Development: OA projects support the principle of "open development"[1], a model where the software is both *by* and *for* the community.  More concretely, these projects try to work in publicly-visible spaces, encouraging discussion and participation from their public user community on both code and project-level decisions.
[1] Tollerud, E., “Sustaining Community-Driven Software for Astronomy in the 2020s”, vol. 51, no. 7, 2019.

> 3. Open Science: OA projects support open and accessible data and reproducible science both through good data management practices and effective open software.

Author: ```suggestion
3. Open Science: OA projects supports reproducible science through the use of open and accessible data both through good data management practices and effective open software.
```

OpenAstronomy is a collaboration between open source astronomy and astrophysics
projects to share resources, ideas, and to improve code.

## Principles of OpenAstronomy

The core principles of OpenAstronomy (OA) include:

1. Open Source: OA projects are committed to their software being Open Source[1]. This promotes accessibility, reliability, and scientific reproducibility.
2. Open Development: OA projects support the principle of Open Development[2], a model where the software is both *by* and *for* the community.  More concretely, these projects try to work in publicly-visible spaces, encouraging discussion and participation from their public user community on both code and project-level decisions.

> 3. Open Science: OA projects supports reproducible science through the use of open and accessible data both through good data management practices and effective open software.

Reviewer: ```suggestion
3. Open Science: OA projects support reproducible science through the use of open and accessible data both through good data management practices and effective open software.
```
